cdap-app-fabric/src/test/java/co/cask/cdap/metadata/LineageAdminTest.java

```diff
@@ -525,6 +649,26 @@ private void addRuns(Store store, ProgramRunId... runs) {
     }
   }
 
+  /** Adds runs which have workflows associated with them
+   *
+   * @param store store instance
+   * @param workflowName name of the workflow
+   * @param workflowRunId run ID associated with all program runs
+   * @param runs list ofo runs to be added
+   */
+  private void addWorkflowRuns(Store store, String workflowName, String workflowRunId, ProgramRunId... runs) {
+    Map<String, String> workflowIDMap = new HashMap<>();
+    Map<String, String> emptyMap = ImmutableMap.of();
+    workflowIDMap.put("workflowName", workflowName);
```

Comment:
does it make sense to utilize `ProgramOptionConstants.WORKFLOW_NODE_ID` where possible?